accounts/src/main/kotlin/pl/edu/pw/ia/accounts/application/AccountController.kt

```diff
@@ -54,6 +55,9 @@ class AccountControllerImpl(
 	override fun createAccount(
 		@RequestBody request: CreateAccountRequest
 	): Mono<IdResponse> {
+		if (request.operationType != "CREATE" || request.resourceType != "USER") {
```

Comment:
this should be an enum, but I don't know all the possible types, 

also this should be on the calle side but with current setup it's impossible.

Treat it as a temporary solution.